__mocks__/handlers/users.handler.js

```diff
@@ -12,46 +12,49 @@ const usersHandler = [
         let prevId;
 
         let filteredUsers = usersData.filter((userObj) => {
-            return (!searchParamToMatch || userObj.username.startsWith(searchParamToMatch));
+            return (
+                !searchParamToMatch ||
+                userObj.username.startsWith(searchParamToMatch)
+            );
         });
 
-        if(nextIdToMatch) {
-            const index = filteredUsers.findIndex(userObj => {
-                return (
-                    userObj.id === nextIdToMatch
-                    );   
-                });
-                const startIndex = index + 1;
-                const endIndex = startIndex + size;
-                filteredUsers = filteredUsers.slice(startIndex, endIndex);
-            } else if (prevIdToMatch) {
-            const index = filteredUsers.findIndex(userObj => {
-                return (
-                    userObj.id === nextIdToMatch
-                );   
-               });
-            
+        if (nextIdToMatch) {
+            const index = filteredUsers.findIndex((userObj) => {
+                return userObj.id === nextIdToMatch;
+            });
+            const startIndex = index + 1;
+            const endIndex = startIndex + size;
+            filteredUsers = filteredUsers.slice(startIndex, endIndex);
+        } else if (prevIdToMatch) {
+            const index = filteredUsers.findIndex((userObj) => {
+                return userObj.id === nextIdToMatch;
+            });
+
             const startIndex = index - filteredUsers.length;
             const endIndex = startIndex + size;
             filteredUsers = filteredUsers.slice(startIndex, endIndex);
-        } else {
-            filteredUsers = filteredUsers.slice(0,size);
+        } else if (size) {
+            filteredUsers = filteredUsers.slice(0, size);
         }
         nextId = filteredUsers[filteredUsers.length - 1].id ?? '';
         prevId = filteredUsers[0].id ?? '';
 
         return res(
             ctx.status(200),
             ctx.json({
-            message: 'Users returned successfully!',
-            users: filteredUsers,
-            links: {
-                next: nextId ? `/users?size=${size}&next=${nextId}&search=${searchParamToMatch}` : '',
-                prev: prevId ? `/users?size=${size}&prev=${prevId}&search=${searchParamToMatch}` : '',   
-            }
+                message: 'Users returned successfully!',
+                users: filteredUsers,
+                links: {
+                    next: nextId
+                        ? `/users?size=${size}&next=${nextId}&search=${searchParamToMatch}`
```

Comment:
Nit: can create a const variable and then only append next or prev instead of typing the entire URL twice.